modules/line.py

```diff
@@ -0,0 +1,23 @@
+from dotenv import load_dotenv
+from dataclasses import dataclass, field
+from typing import Dict
+
+import os
+import requests
+
+load_dotenv(override=True)
+
+@dataclass
+class Line:
+    headers: Dict[str, str] = field(default_factory=lambda: {
```

Comment:
Not sure but I remember it's work XD So that we can remove line #3 `from typing import Dict`
```suggestion
    headers: dict[str, str] = field(default_factory=lambda: {
```